maintainers/README.md

```diff
@@ -2,7 +2,22 @@
 
 ## Motivation
 
-The goal of the team is to help other people to contribute to Nix.
+In order to attract more maintainers and thereby scale the development process to match user needs, the main goal of this team is to improve the Nix contributor experience.
+
+### Objectives
+
+- It is obvious what is worthwhile to work on.
+- It is easy to find the right place in the code to make a change.
+- It is clear what is expected of a pull request.
+- It is predictable how to get a change merged and released.
```

Comment:
I think this is missing a few important items: 
- The team should take responsibility for making sure that the codebase stays in a good shape (beyond just “enabling other people to do the job”). In particular, that means handling a few key aspects of the development like:
    - Release management
    - Making sure that _someone_ is available to fix the most critical bugs
- There's an “external communication” aspect that we haven't been keeping up too much but is important (telling the world how great contributing to and using Nix is)
- The team should also set the expectations for users (roadmap & regular releases in particular)

Reply:
- Architecture & Design!

Reply:
More clearly separated objectives from tasks.